app/views/layouts/_registers-in-numbers.html.haml

```diff
@@ -1,19 +1,21 @@
-%aside{ class: "#{current_page?(about_path) ? 'registers-in-numbers--about-page' : 'registers-in-numbers'}"}
-  %h2{ class: "#{current_page?(about_path) ? 'govuk-heading-s' : 'govuk-heading-m'}"} GOV.UK Registers in numbers
-  %ul
-    - if current_page?(about_path)
-      %li{class: 'registers-in-numbers__item govuk-!-margin-bottom-4'}
-        %span{role: "presentation", class: 'govuk-heading-xl govuk-!-margin-bottom-0'} 800k
-        %a{role: "presentation", class: 'data-item govuk-body-s govuk-!-margin-bottom-0', href: "https://www.gov.uk/performance/govuk-registers"}total API requests
-    %li{class: 'registers-in-numbers__item govuk-!-margin-bottom-4'}
-      -# = link_to registers_path do
-      %span{role: "presentation", class: "#{current_page?(about_path) ? 'govuk-heading-xl' : 'govuk-!-font-size-80 govuk-!-font-weight-bold'} govuk-!-margin-bottom-0"}= Register.available_count
-      %span{role: "presentation", class: "#{current_page?(about_path) ? 'govuk-body-s' : 'govuk-body'} govuk-!-margin-bottom-0 data-item"} government registers available
-    %li{class: 'registers-in-numbers__item govuk-!-margin-bottom-4'}
-      -# = link_to registers_path do
-      %span{role: "presentation", class: "#{current_page?(about_path) ? 'govuk-heading-xl' : 'govuk-!-font-size-80 govuk-!-font-weight-bold'} govuk-!-margin-bottom-0"}= Register.organisation_count
-      %span{role: "presentation", class: "#{current_page?(about_path) ? 'govuk-body-s' : 'govuk-body'} govuk-!-margin-bottom-0 data-item"} government organisations managing&nbsp;registers
-    %li{class: 'registers-in-numbers__item govuk-!-margin-bottom-4'}
-      -# = link_to services_using_registers_path do
-      %span{role: "presentation", class: "#{current_page?(about_path) ? 'govuk-heading-xl' : 'govuk-!-font-size-80 govuk-!-font-weight-bold'} govuk-!-margin-bottom-0"} 8
-      %span{role: "presentation", class: "#{current_page?(about_path) ? 'govuk-body-s' : 'govuk-body'} govuk-!-margin-bottom-0 data-item"} government services using&nbsp;registers
+- list_presentation_class = defined?(presentation).nil? ? '' : " registers-in-numbers--#{presentation}"
+- heading_border_class = defined?(border).nil? ? '' : " registers-!-border-top"
+- heading_size = defined?(heading).nil? ? 'govuk-heading-m' : 'govuk-heading-s'
+
+%h2{ class: "#{heading_size} #{heading_border_class} govuk-!-padding-top-4"} GOV.UK Registers in numbers
+%ul{class: "registers-in-numbers#{list_presentation_class}" }
+  %li{class: 'registers-in-numbers__item'}
+    %span{role: 'presentation', class: 'registers-in-numbers__number'} 800k
+    %a{role: 'presentation', class: 'registers-in-numbers__caption', href: 'https://www.gov.uk/performance/govuk-registers'}total API requests
+  %li{class: 'registers-in-numbers__item'}
+    -# = link_to registers_path do
```

Comment:
If this is commented out can we remove it now?

Reply:
Removed 👍